.github/actions/changes/action.yml

```diff
@@ -148,4 +148,4 @@ runs:
         type: ${{ inputs.type }}
         init: true
         mergeable: ${{ steps.pr.outputs.source_branch == 'main' || steps.pr.outputs.mergeable }}
-        commit_status: ${{ inputs.commit_status }}
+        commit_status: ${{ inputs.type == 'visual' && steps.should_run.outputs.should_run == 'false' || inputs.commit_status }}
```

Comment:
We have to do this here. If the job doesn't need to run we are making a status with the same name (so we can pass the required check), if it _does_ need to run we don't make a status and let the chromatic action handle it.